editor/src/clj/editor/editor_extensions.clj

```diff
@@ -357,6 +359,15 @@
       (when (and resource (resource/exists? resource) (resource/openable? resource))
         (open-resource! resource)))))
 
+(def ext-open-url-fn
+  (rt/suspendable-lua-fn open-url [{:keys [rt]} lua-string]
+    (let [s (rt/->clj rt coerce/string lua-string)]
+      (future/io
+        (try
+          (.browse ui/desktop (URI. s))
```

Comment:
I didn't use `ui/open-url` because I need to raise an error on failure to open. I checked this changeset on Linux and it does not block the editor (`java.awt.Desktop` and JavaFX...)